kumascript/macros/CSSSyntax.ejs

```diff
@@ -430,7 +430,8 @@ if (name === "preview-wiki-content") {
 }
 
 if (!formattedSyntax) {
-    throw new Error(`No syntax found in DB for '${name}'`);
+  formattedSyntax = "<span style=\"color:red;\">" +
+    `No syntax found in DB for '${name}'` + "</span>";
```

Comment:
Are you sure inline block of `<span>` is correct use here? What about a `<div class="notecard warning">` instead so there's consistency in alerts?

Reply:
I copied the style used in CSSInfo, but let's change this. I can even change the one in CSSInfo at the same time to have full consistency.

Let's do this.

Reply:
Fixed (I decided to leave CSSInfo out of this finally).

It looks like this:
<img width="721" alt="Screenshot 2021-08-17 at 18 36 27" src="https://user-images.githubusercontent.com/1466293/129765954-aabd390e-f59e-4f36-8c74-c35e3b8f42aa.png">